readme.md

```diff
@@ -85,6 +85,73 @@ grammar.
 You can also join the [Discord for development discussion]((https://discord.gg/5QTexBU))
 if you are unsure of anything.
 
-## License
+# Extracting asset files from original game
+
+As mentioned before, the original game assets are required in order to run
+openblack. Since these data are copyrighted, **we won't provide them**, so please
+don't ask. Here is a guide for extracting them from the original game.
+
+## Prerequisites
+
+You will need the following:
+
+- The original installer (usually on a CD)
+- Official patch 1.10
+- Official patch 1.2
+- Not needed (but will not compromise the extraction): Unofficial patch 1.42
+
+Patches can be found on fan sites such as
+[Black & White REALM](https://www.bwrealm.com/official-files-patches-1)
+
+## Installation
+
+### On Windows
+
+1. Install the game from the installer, remember the installation directory
+(`C:\\Program Files (x86)\\Lionhead Studios\\Black & White\\` is the default)
+2. Install the official patch 1.10 (in the same directory)
+3. Install the official patch 1.2 (in the same directory)
+
+### On Linux
```

Comment:
The instructions are the same for MacOS, please add that.

Reply:
@bwrsandman : where is the wine directory on macos? is it in ~/.wine/ too?

Reply:
According to some docs, it is, I pushed the new version.